### PR TITLE
Css tweaks

### PR DESCRIPTION
**Pull request by kofimeighan**

### CODE REVIEW

File: portfolio/src/main/webapp/css/custom.css [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+ * Copyright 2011-2020 The Bootstrap Authors
+ * Copyright 2011-2020 Twitter, Inc.
+ * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
+ */
+/* button  - #006600
+Button hover -  #004d00
+Button outline - #003300
+Text color -  #006600
+Gray - #6c757d
+*/
```
Comment (briafassler9) [inline portfolio/src/main/webapp/css/custom.css]: Is this necessary to keep since it's commented out?

**Comment by Jaxobc** [comment]
Apart from Bria's comment, it looks good to me.

File: portfolio/src/main/webapp/css/custom.css [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+  background-image: url('../assets/img/header.jpg');
+  background-size: cover;
+}
+
+.btn-primary {
+  color: #fff;
+  background-color: #006600;
+  border-color: #006600;
+}
+.btn-primary:hover {
```
Comment (briafassler9) [inline portfolio/src/main/webapp/css/custom.css]: Add space between each class

File: portfolio/src/main/webapp/css/custom.css [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+  background-color: #006600;
+  border-color: #006600;
+}
+.btn-primary:not(:disabled):not(.disabled):active,
+.btn-primary:not(:disabled):not(.disabled).active,
+.show > .btn-primary.dropdown-toggle {
+  color: #fff;
+  background-color: #004d00;
+  border-color: #006600;
+}
```
Comment (briafassler9) [inline portfolio/src/main/webapp/css/custom.css]: Add space between each class for consistency

File: portfolio/src/main/webapp/css/custom.css (hunk 1 above)
Comment (kofimeighan) [reply portfolio/src/main/webapp/css/custom.css]: deleted

File: portfolio/src/main/webapp/css/custom.css (hunk 3 above)
Comment (kofimeighan) [reply portfolio/src/main/webapp/css/custom.css]: added

**Review by briafassler9 — APPROVED** [review]

**Review by kennzoid — APPROVED** [review]
Great PR already. My only nit is that the description should be more descriptive.

Maybe something like:
"Replaced placeholder photos with actual martyr photos and updated some of the CSS."

**Pull request merged**